tests/e2e/config/bootstrap.js

```diff
@@ -152,10 +152,8 @@ async function observeRestResponse( res ) {
 		// The response may fail to resolve if the test ends before it completes.
 		try {
 			args.push( await res.text() );
+			console.log( ...args ); // eslint-disable-line no-console
```

Comment:
What's the thinking here? Only logging when the test has not ended yet?

Reply:
Jest will fail the test if something is logged after the test is over, so if that happens for some reason then we just swallow the error to prevent that from happening.